Dockerfile

```diff
@@ -46,9 +46,8 @@ COPY . ${RAILS_ROOT}
 FROM ${BASE}
 
 ENV LANG en_US.UTF-8
-
 RUN apt-get update -qq \
-  && apt-get install -y libjemalloc2 postgresql-client tzdata libv8-dev curl default-jre \
```

Comment:
Why the `postgresql-client` change here along with moving it down?

Reply:
The pg_restore requires version compatibility, which involves the whole curl and unpack situation in order to get the exact right version for the restore and the linux distro. It was complicated so I moved it into its own step.